dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +684,208 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	u := uuid.Must(uuid.NewV4()).String()[:8]
+	datastoreNamespace := "golang-samples" + u
+	datastoreKind := "task"
+
+	if err := inspectDataStoreSendToScc(&buf, tc.ProjectID, datastoreNamespace, datastoreKind); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "Job created successfully:"; !strings.Contains(got, want) {
+		t.Errorf("InspectBigQuerySendToScc got %q, want %q", got, want)
+	}
+}
+
+const (
+	dataSetID = "dlp_test_dataset"
+	tableID   = "dlp_inspect_test_table_table_id"
+)
+
+func TestInspectBigQuerySendToScc(t *testing.T) {
```

Comment:
issue: one test per file, please.

See https://googlecloudplatform.github.io/samples-style-guide/#dedicated-testing-per-sample

Reply:
Waiting for your reply on https://github.com/GoogleCloudPlatform/golang-samples/pull/3242#discussion_r1313028235

Reply:
Hey @telpirion Hi, I have split the test for this particular sample into a separate file.